apps/rush-lib/src/cli/actions/InstallAction.ts

```diff
@@ -105,9 +105,9 @@ export default class InstallAction extends BaseRushAction {
 
       if (!installManager.createTempModulesAndCheckShrinkwrap(shrinkwrapFile, installType !== InstallType.Normal)) {
         console.log('');
-        console.log(colors.red('You need to run "rush generate" to update your NPM shrinkwrap file.'));
+        console.log(colors.red('You need to run "rush generate" to update your shrinkwrap file.'));
         process.exit(1);
-        return;
+        return Promise.resolve();
```

Comment:
>return Promise.resolve(); [](start = 8, length = 25)

Since exit() returns `never`, it's interesting that the compiler allowed a return statement here at all.  Maybe we're missing a strict option